laravel-app/resources/views/profile/show.blade.php

```diff
@@ -7,13 +7,18 @@
 <div class="profile-page">
 	<div class="container-fluid">
 		<div class="row justify-content-center post-card p-5">
+			@if (session('success'))
+				<div class="alert alert-success">
+					{{ session('success') }}
+				</div>
+			@endif
 			<div class="col-md-5 text-center">
 				<div class="row justify-content-center text-center">
 					<div class="col-auto">
 						@if ($user->image_path === "assets/images/user-solid.svg")
 							<div class="profile-letter">
 								<div class="letter-bg">
-									{{ substr($user->full_name, 0, 1) }}
+									{{ ucfirst(substr($authUser->full_name, 0, 1)) }}
```

Comment:
I think this is not necessary to be added as an accessor to the user model since it was only used twice throughout the code base. This might change in the future, and I might add the accessor if it seems necessary by then. Thank you. ❤️

Reply:
Hmmm, if you think so. Marked this as resolve then.